tools/docker/builder/ubuntu/bionic/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:18.04
```

Comment:
Why not use the same Dockerfile with a build argument controlling the version?

Reply:
Although some package names might be the same between the 2 versions, others aren't.
So IMO it's really easier and clearer to have 2 different Dockerfile for the 2 different versions from the start.

Reply:
Yes, this is the point for this change.